Packs/CortexAttackSurfaceManagement/Scripts/SnmpDetection/SnmpDetection.yml

```diff
@@ -0,0 +1,20 @@
+args:
+- description: IP address.
+  name: IP
```

Comment:
```suggestion
  name: ip_address
```

Reply:
Hello @adi88d, thank you for the suggestion. Updated the files to have [ip_address](https://github.com/demisto/content/blob/4a6e04b4e0fc5d3a1c73748435bde9fba4f2d281/Packs/CortexAttackSurfaceManagement/Scripts/SnmpDetection/SnmpDetection.yml)